### PR TITLE
Make testgrid-config-upload job use bootstrap

### LINKED ISSUE #1039: testgrid-config-upload job is failing
got `AccessDeniedException: 403 Forbidden` now

open this to track the issue.

### PR DESCRIPTION
**Pull request by krzyzacy**
hope to fix #1039 

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1047)

### CODE REVIEW

**Comment by fejta** [comment]
thanks!

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -845,7 +845,7 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/kubernetes-e2e-gce-enormous-teardown
 - name: kubernetes-e2e-gke-gci-test-release-1.2
   gcs_prefix: kubernetes-jenkins/logs/kubernetes-e2e-gke-gci-test-release-1.2
-- name: testgrid-config-upload
+- name: maintainance-ci-testgrid-config-upload
```
Comment (fejta) [inline testgrid/config/config.yaml]: Why do we need the maintenance-ci prefix?

File: testgrid/config/config.yaml [hunk 2]
```diff
@@ -845,7 +845,7 @@ test_groups:
...
   gcs_prefix: kubernetes-jenkins/logs/testgrid-config-upload
```
Comment (fejta) [inline testgrid/config/config.yaml]: This needs to change, right?

File: testgrid/config/config.yaml (hunk 1 above)
Comment (krzyzacy) [inline testgrid/config/config.yaml]: meh... I was referencing maintenance-ci-clean-project... I'll keep the name then

**Comment by krzyzacy** [comment]
JESUS I cannot spell maintenance

**Comment by fejta** [comment]
lol :laughing:

**Pull request merged**